internal/test/test_backend.go

```diff
@@ -122,7 +125,7 @@ func echo(rw http.ResponseWriter, req *http.Request) {
 	// Clear deadlines
 	conn.SetDeadline(time.Time{})
 
-	_, err = conn.Write([]byte("HTTP/1.1 101 Switching Protocols\r\nUpgrade: websocket\r\nConnection: Upgrade\r\n\r\n"))
+	_, _ = conn.Write([]byte("HTTP/1.1 101 Switching Protocols\r\nUpgrade: websocket\r\nConnection: Upgrade\r\n\r\n"))
```

Comment:
Discarding the error is the wrong solution if a reference or handling is missing. Error check should be added instead of removal.